riscv/src/elf/mod.rs

```diff
@@ -190,6 +191,8 @@ fn load_elf(file_name: &Path) -> ElfProgram {
         text_labels: referenced_text_addrs,
         instructions: lifted_text_sections,
         entry_point: elf.entry as u32,
+        // TODO: properly load these from the ELF instead of hardcoding them.
+        prover_data_bounds: (0x10000000, 0x20000000),
```

Comment:
@lvella we still need to do this

Reply:
here?

Reply:
Let me start a PR now. Can be merged before or after.